arch/arm/boot/dts/overlays/vc4-kms-dpi-geekworm-mzp280-overlay.dts

```diff
@@ -0,0 +1,70 @@
+/*
+ * vc4-kms-dpi-at056tn53v1-overlay.dts
+ */
+
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			panel: panel {
+				compatible = "geekworm,mzp280";
+				rotation = <0>;
+
+				port {
+					panel_in: endpoint {
+						remote-endpoint = <&dpi_out>;
+					};
+				};
+			};
+		};
+	};
+
+	fragment@1 {
+		target = <&dpi>;
+		__overlay__  {
+			status = "okay";
+
+			pinctrl-names = "default";
+			pinctrl-0 = <&dpi_16bit_cpadhi_gpio0>;
+
+			port {
+				dpi_out: endpoint {
+					remote-endpoint = <&panel_in>;
+				};
+			};
+		};
+	};
+
+	fragment@2 {
+		target = <&gpio>;
+		__overlay__ {
+			dpi_16bit_cpadhi_gpio0: dpi_16bit_cpadhi_gpio0 {
+				brcm,pins = <0 1 2 3 4 5 6 7 8
+					     12 13 14 15 16 17
+					     20 21 22 23 24>;
+				brcm,function = <6>; /* alt2 */
+				brcm,pull = <0>; /* no pull */
+			};
+		};
+	};
+
+	fragment@3 {
+		target = <&spi0>;
+		__overlay__ {
+			pinctrl-0 = <&spi0_pins>;
+			/delete-property/ cs-gpios;
```

Comment:
Unfortunately this doesn't do what you think - overlays can't delete properties or nodes.